install/setup.bash

```diff
@@ -24,9 +24,16 @@ _ament_prefix_path_add() {
     fi
 }
 
+_pythonpath_add() {
+    if [ -d "$1" ] && [[ ":$PYTHONPATH:" != *":$1:"* ]]; then
+        export PYTHONPATH="$1${PYTHONPATH:+":$PYTHONPATH"}"
+    fi
+}
+
 _path_add "${_INSTALL_PATH}/bin"
 _ld_library_path_add "${_INSTALL_PATH}/lib"
 _ament_prefix_path_add "${_INSTALL_PATH}"
+_pythonpath_add "${_INSTALL_PATH}/lib/python3.6/site-packages"
```

Comment:
Are we pinned to 3.6 in the ubuntu_setup script?

Reply:
Ok I changed this to use the same thing ament is using